docs/source/using/experimental-features.adoc

```diff
@@ -73,6 +73,11 @@ If you prefer to use dynamic assignment, you can use:
 $ minishift ip --set-dhcp
 ----
 
+[NOTE]
+====
+Use `minishift config set static-ip true` to set static ip automatically for supported hypervsiors.
```

Comment:
@praveenkumar You need to change this now as we are enabling this by default. Also, this is not an experimental feature as we are enabling it by default. So we need to move the document to https://docs.okd.io/latest/minishift/using/static-ip.html. But that would need some restructuring of https://docs.okd.io/latest/minishift/using/static-ip.html which we can do post merging this PR and after talking to @robin-owen